docs/concepts/cluster-administration/federation-service-discovery.md

```diff
@@ -189,32 +189,32 @@ nameServers:
-nginx.mynamespace.myfederation.svc.us-central1.example.com.         A    180     104.197.247.191, 104.197.244.180, 104.197.245.170
-nginx.mynamespace.myfederation.svc.asia-east1-a.example.com.       A    180     130.211.57.243
-nginx.mynamespace.myfederation.svc.asia-east1-b.example.com.  CNAME 180     nginx.mynamespace.myfederation.svc.asia-east1.example.com.
-nginx.mynamespace.myfederation.svc.asia-east1-c.example.com.       A    180     130.211.56.221
-nginx.mynamespace.myfederation.svc.asia-east1.example.com.           A    180     130.211.57.243, 130.211.56.221
-nginx.mynamespace.myfederation.svc.europe-west1.example.com.  CNAME    180   nginx.mynamespace.myfederation.svc.example.com.
-nginx.mynamespace.myfederation.svc.europe-west1-d.example.com.  CNAME   180     nginx.mynamespace.myfederation.svc.europe-west1.example.com.
+NAME                                                            TYPE      TTL     DATA
+example.com.                                                    NS        21600   ns-cloud-e1.googledomains.com., ns-cloud-e2.googledomains.com.
+example.com.                                                    OA        21600   ns-cloud-e1.googledomains.com. cloud-dns-hostmaster.google.com. 1 21600 3600 1209600 300
+nginx.mynamespace.myfederation.svc.example.com.                 A         180     104.197.246.190, 130.211.57.243, 104.196.14.231, 104.199.136.89,...
+nginx.mynamespace.myfederation.svc.us-central1-a.example.com.   A         180     104.197.247.191
+nginx.mynamespace.myfederation.svc.us-central1-b.example.com.   A         180     104.197.244.180
+nginx.mynamespace.myfederation.svc.us-central1-c.example.com.   A         180     104.197.245.170
+nginx.mynamespace.myfederation.svc.us-central1-f.example.com.   CNAME     180     nginx.mynamespace.myfederation.svc.us-central1.example.com.
+nginx.mynamespace.myfederation.svc.us-central1.example.com.     A         180     104.197.247.191, 104.197.244.180, 104.197.245.170
+nginx.mynamespace.myfederation.svc.asia-east1-a.example.com.    A         180     130.211.57.243
+nginx.mynamespace.myfederation.svc.asia-east1-b.example.com.    CNAME     180     nginx.mynamespace.myfederation.svc.asia-east1.example.com.
+nginx.mynamespace.myfederation.svc.asia-east1-c.example.com.    A         180     130.211.56.221
+nginx.mynamespace.myfederation.svc.asia-east1.example.com.      A         180     130.211.57.243, 130.211.56.221
+nginx.mynamespace.myfederation.svc.europe-west1.example.com.    CNAME     180     nginx.mynamespace.myfederation.svc.example.com.
+nginx.mynamespace.myfederation.svc.europe-west1-d.example.com.  CNAME     180     nginx.mynamespace.myfederation.svc.europe-west1.example.com.
 ... etc.
 ```
 
 Note: If your Federation is configured to use AWS Route53, you can use one of the equivalent AWS tools, for example:
 
 ``` shell
-$aws route53 list-hosted-zones
+$ aws route53 list-hosted-zones
 ```
 and
```

Comment:
I think you need an extra newline after "and". Currently, the shell command isn't rendering correctly:

![screen shot 2017-04-27 at 3 16 43 pm](https://cloud.githubusercontent.com/assets/24798125/25506667/9cfd545a-2b5c-11e7-95eb-1c75ca480f0c.png)